hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/fs/viewfs/TestViewFileSystemHdfs.java

```diff
@@ -219,6 +221,9 @@ public void testTrashRootsAfterEncryptionZoneDeletion() throws Exception {
     DFSTestUtil.verifyDelete(shell, fsTarget, zone, true);
     assertTrue("ViewFileSystem trash roots should include EZ zone trash",
         (fsView.getTrashRoots(true).size() == 2));
+    } finally {
+      DFSTestUtil.deleteKey("test_key", cluster);
```

Comment:
Can we do this in a separate JIRA?

Reply:
If i recall correctly I needed to add this because a test was failing. My new tests also use test keys, and i don't think this cleanup is correct between tests.
I can try and remove it thought and double check if you prefer (It just takes a long time for tests to run), but happy to do it